SiteAsii/src/app/about/about.component.html

```diff
@@ -56,7 +56,7 @@ <h2>Despre noi...</h2>
   </div>
   <div class="question">
     <h3>Eşti curios din cine este alcătuită A.S.I.I. ?</h3>
-    <button mat-stroked-button class="view-button">Vezi membrii ASII</button>
+    <button mat-stroked-button class="view-button" routerLink="/membrii">Vezi membrii ASII</button>
```

Comment:
Remove the routerLink, it's not necessary for now

Reply:
Asta nu e facuta de mine